test/Interop/Cxx/class/Inputs/constructors.h

```diff
@@ -307,15 +307,15 @@ __attribute__((swift_attr("release:Release10"))) DeletedCtor {
 
 struct __attribute__((swift_attr("import_reference")))
 __attribute__((swift_attr("retain:Retain11")))
-__attribute__((swift_attr("release:Release11"))) CtorWithDefaultArg {
+__attribute__((swift_attr("release:Release11"))) CtorWithDefaultArg { // expected-note {{'init(_:)' declared here}}
```

Comment:
Where are these notes coming from? I think we should not emit them as the user never actually created these annotations. That being said, I am OK doing this in a follow-up PR.